PUBLISH.md

```diff
@@ -0,0 +1,88 @@
+# Publishing packages
+
+
+## CXL Registry
+
+We maintain custom npm registry: https://npm.cxl.co and publish there Aybolit packages.
+
+It is important to check if there's `.npmrc` file on main Aybolit directory with following entry:
+
+```
+@conversionxl:registry=https://npm.cxl.co
+```
+
+## Log in
+
+In order to publish packages you need to log in to npm registry.
+
+```
+npm login --registry=https://npm.cxl.co
+```
+
+## Install dependencies and build
+
+Run following commands in main repository directory:
+
+```
+yarn
+yarn build
+yarn dist
+```
+
+## Publish package
+
+**Important:** before performing following steps make sure your workdir is clean and contains only necessary files and directories. Clean it up from everyting else as publish everything sitting in the package directory.
+
+In following steps `cxl-ui` package is used but you can replace it with name of the other package.
+
+
+### Steps
+
+1. `cd packages/cxl-ui`
+2. Edit package version in `package.json`.
+
+  ```diff
+  - "version": "1.1.0-alpha.2",
+  + "version": "1.1.0-alpha.3",
+  ```
+
+3. Save and commit it.
+4. `yarn publish`
+5. Go through all steps. When asked about version, use value you just placed in `package.json` eg. `1.1.0-alpha.3`
```

Comment:
@anoblet @saas786 I was also wondering abut steps 2 and 5. yarn publish wizard suggest bumped version, but I don't know if it modifies `package.json` in current workdir or just bumps it in the registry. Maybe doing manual edits is not necessary.

Reply:
> @anoblet @saas786 I was also wondering abut steps 2 and 5. yarn publish wizard suggest bumped version, but I don't know if it modifies `package.json` in current workdir or just bumps it in the registry. Maybe doing manual edits is not necessary.

If it finds changes in the package, I think it does bump the package version automatically, without you manually changing it.

Reply:
I tested recently and indeed, it bumps the package version automatically. But after bumping version I received and error and eventually edited it manually. Maybe over time I'll find out what the problem was